src/OpenLoco/src/GameCommands/CreateWall.cpp

```diff
@@ -0,0 +1,278 @@
+#include "Economy/Economy.h"
+#include "Economy/Expenditures.h"
+#include "GameCommands.h"
+#include "Localisation/StringIds.h"
+#include "Map/QuarterTile.h"
+#include "Map/SurfaceElement.h"
+#include "Map/TileClearance.h"
+#include "Map/TileManager.h"
+#include "Map/TreeElement.h"
+#include "Map/WallElement.h"
+#include "Objects/LandObject.h"
+#include "Objects/ObjectManager.h"
+#include "Objects/WallObject.h"
+#include "OpenLoco.h"
+#include "S5/S5.h"
+#include "Scenario.h"
+#include "Ui/WindowManager.h"
+#include "ViewportManager.h"
+#include "World/TownManager.h"
+#include <OpenLoco/Core/EnumFlags.hpp>
+
+using namespace OpenLoco::Interop;
+using namespace OpenLoco::World;
+
+namespace OpenLoco::GameCommands
+{
+    // clang-format off
+    enum class EdgeSlope : uint8_t
+    {
+        none              = 0U,
+        elevated          = (1 << 0),
+        upwards           = (1 << 6),
+        downwards         = (1 << 7),
```

Comment:
1U please, signed integer shifts are pretty bad especially when you get sign bit.